.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,84 @@
+name: "CodeQL"
+
+on:
+  # Run the analysis once every 24 hours. The actual time does not matter
+  # that much, start with a time that allows for easier troubleshooting.
+  schedule:
+    - cron: '00 22 * * *'
+  push:
+    branches: ['ci-experiment-codeql']
```

Comment:
main?

Reply:
This is temporary.  I will remove the whole `push` section.

Reply:
Done now.